src/MessengerShareButton.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+
+import assert from 'assert';
+
+import objectToGetParams from './utils/objectToGetParams';
+import createShareButton from './utils/createShareButton';
+
+function messengerLink(url, { redirectUri, appId }) {
+  assert(url, 'messenger.url');
+
+  return 'http://www.facebook.com/dialog/send' + objectToGetParams({
+    app_id: appId,
+    link: url,
+    redirect_uri: redirectUri,
+  });
+}
+
+const MessengerShareButton = createShareButton('messenger', messengerLink, props => ({
```

Comment:
`MessengerShareButton` Component couldn't understand, which social media messenger you want to share

Reply:
I will change to ```FacebookMessengerShareButton```, thank for comment.